versions/3.0.md

```diff
@@ -96,8 +96,7 @@ The available status codes are defined by [RFC7231](http://tools.ietf.org/html/r
 
 ### Format
 
-The files describing the RESTful API in accordance with this specification consist of JSON objects and conform to the JSON standards. An OAS file written in
-YAML, a superset of JSON, also complies with this specification. 
+An API description that conforms to the OpenAPI Specification is itself a JSON object, which may be represented either in JSON or YAML format.
```

Comment:
If we adopt my suggestion on #1154 then this would be "An OpenAPI definition that conforms to the OpenAPI Specification...." (rather than *An API description*)